go/libkb/id_table.go

```diff
@@ -769,6 +772,53 @@ func (s *PerUserKeyChainLink) ToPerUserKey() keybase1.PerUserKey {
 	}
 }
 
+//-------------------------------------
+
+// VerifyReverseSig checks a SibkeyChainLink's reverse signature using the ComputedKeyFamily provided.
+func (s *PerUserKeyChainLink) VerifyReverseSig(_ ComputedKeyFamily) (err error) {
```

Comment:
Sure, I guess that would use https://github.com/keybase/keybase-test-vectors . Is that `iced/cson -> json -> go`?